README.md

```diff
@@ -351,3 +391,95 @@ $ npm run lint
 ## Github Actions
 
 All pushes to this repo will be checked with `npm test` and `npm lint`.
+
+## Git Workflow
+
+Our branches (in order of stability are):
+
+| Branch     | Environment | AWS Account    | Link To Application                                                                        |
+| :--------- | :---------- | :------------- | :----------------------------------------------------------------------------------------- |
+| main       | development |                | [localhost:3000](http://localhost:3000)                                                    |
+| qa         | qa          | nypl-dams-dev  | [https://qa-new-digitalcollections.nypl.org/](https://qa-new-digitalcollections.nypl.org/) |
+| production | production  | nypl-dams-prod | [https://new-digitalcollections.nypl.org/](https://new-digitalcollections.nypl.org/)       |
+
+## Contributing
+
+1.  Feature branches are cut from `main` using
+
+    `git checkout -b feature/DR-123/feature-name`.
+
+2.  Add your changes to the CHANGELOG.md file under `## [Unreleased]`. See [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) for formatting docs.
+3.  Create a pull request of the `feature` branch _into_ `main`.
+4.  After a feature branch has approved accessibility review and VQA, merge `feature` branch into `main`, then merge `main` branch into `qa` for testing.
+5.  After QA has tested and approved cut a `release` branch off of `qa` using our Release Naming Strategy. Please see Release Naming Strategy for details on how we plan to number our releases during the initial rollout of this project.
+
+`git checkout -b release/x.y.z`.
+
+(see tagged releases or changelog for last version)
+
+6. Update CHANGELOG.md in release branch by moving updates from unreleased into the new release section.
+
+   ie. `## [2.4.13] - 04-21-2023`
+
+7. Commit and push changes to release branch.
+8. Merge `release` branch to `production` and push the changes. This will deploy through Travis if all goes well.
+9. Immediately after merging release to production, finish the release and backmerge release to `qa` and `main`
+
+   `$ git tag -a x.y.z`
+
+   `$ git push origin --tags`
```

Comment:
The 9th bullet point isn't instructions for these commands. So add that you also want to create tags/release on github and this is how it's done. Hope this can be automated at some point.